airbyte-integrations/connectors/source-open-exchange-rates/setup.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
```

Comment:
I think you should explicitely add `pendulum` to the dependencies here

Reply:
Added